docs/usage/shoot_settings/shoot_hibernate.md

```diff
@@ -27,22 +20,22 @@ When a cluster is hibernated, Gardener scales down the worker nodes and the clus
 * The virtual machines running your workload.
 * The resources of the control plane of your cluster.
 
-## What Isn’t Affected by the Hibernation?
+### What Isn’t Affected by the Hibernation?
 
 To scale up everything where it was before hibernation, Gardener doesn’t delete state-related information, that is, information stored in persistent volumes. The cluster state as persistent in `etcd` is also preserved.
 
 ## Hibernate Your Cluster Manually
 
 The `.spec.hibernation.enabled` field specifies whether the cluster needs to be hibernated or not. If the field is set to `true`, the cluster's desired state is to be hibernated. If it is set to `false` or not specified at all, the cluster's desired state is to be awakened.
 
-To hibernate your cluster, you can run the following `kubectl` command:
+To hibernate your cluster, you can do it from the Gardener dashboard or run the following `kubectl` command:
```

Comment:
I would revert the Gardener dashboard part, we don't have to remind every time that every operation is possible from the dashboard as well.